omod/src/main/webapp/pages/login.gsp

```diff
@@ -19,67 +19,74 @@
 
 ${ ui.includeFragment("referenceapplication", "infoAndErrorMessages") }
 
+<% if(showSessionLocations) { %>
 <script type="text/javascript">
     jQuery(function() {
-    	updateSelectedOption = function() {
-	        jQuery('#sessionLocation li').removeClass('selected');
-	        
-			var sessionLocationVal = jQuery('#sessionLocationInput').val();
-	        if(sessionLocationVal != null && sessionLocationVal != "" && sessionLocationVal != 0){
-	            jQuery('#sessionLocation li[value|=' + sessionLocationVal + ']').addClass('selected');
-	        }
-    	};
-    
+        updateSelectedOption = function() {
+            jQuery('#sessionLocation li').removeClass('selected');
+
+            var sessionLocationVal = jQuery('#sessionLocationInput').val();
+            if(sessionLocationVal != null && sessionLocationVal != "" && sessionLocationVal != 0){
+                jQuery('#sessionLocation li[value|=' + sessionLocationVal + ']').addClass('selected');
+            }
+        };
+
         updateSelectedOption();
 
         jQuery('#sessionLocation li').click( function() {
             jQuery('#sessionLocationInput').val(jQuery(this).attr("value"));
             updateSelectedOption();
         });
-		jQuery('#sessionLocation li').focus( function() {
+        jQuery('#sessionLocation li').focus( function() {
```

Comment:
This indentation should be fixed since wrong in the original file -  https://github.com/openmrs/openmrs-module-referenceapplication/blob/master/omod/src/main/webapp/pages/login.gsp#L39